echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/slack/SlackClient.groovy

```diff
@@ -33,9 +34,17 @@ interface SlackClient {
   /**
    * Documentation: https://api.slack.com/incoming-webhooks
    */
-  @POST('/{token}')
+  @POST('/services/{token}')
```

Comment:
Moved the `/services` prefix here from `SLACK_INCOMING_WEBHOOK` in `SlackConfig`, which is now also used for other API calls that don't use that same path.